presto-product-tests/bin/run_on_docker.sh

```diff
@@ -77,7 +77,7 @@ function docker_images_used() {
 function cleanup() {
   stop_application_runner_containers ${ENVIRONMENT}
 
-  if [[ "${LEAVE_CONTAINERS_ALIVE_ON_EXIT:-false}" = "true" ]]; then
+  if [[ "${LEAVE_CONTAINERS_ALIVE_ON_EXIT:-false}" != "true" ]]; then
```

Comment:
This commit is unrelated, right?  Still seems good.  Just want to make sure I understand how these things relate to each other.

Reply:
Yes, unrelated. Popped up when running product tests locally. I just didn't want to run separate PR with this ;)